docs/launcher.md

```diff
@@ -0,0 +1,86 @@
+
+# vLLM Launcher
+
+## Description
+
+This document describes a vLLM launcher to be used in model swapping techniques. The launcher initializes and manages various subsidiary inference vLLM servers and keep an inventory with it's ID's and the model (in addition to other information like the type of accelerator and the GPU number). The launcher does basic vLLM code loading and initialization work (like import modules) of the inference server so that this work does not have to be done at the startup of the inference server process, reducing the startup latency. In future work, the launcher could be extended to sleep and wake up the inference servers that have been initialize by its POST method.
+
+## Launcher Methods and functionalities
+
+The dual-pod controller will use a launcher-specific command to run the launcher. To swap a model in, the controller will issue a POST request (to the launcher) that includes the model reference and the model-specific flags.
+
+Setting up process to create a new vLLM inference instance using POST:
+
+```mermaid
+sequenceDiagram
+    participant Controller
+    participant Launcher
+    participant vLLM instance
+
+    Controller->>Launcher: POST create new instance
+    Launcher-->>vLLM instance: create vLLM instance
```

Comment:
The launcher client needs to know what a reply implies about the state of the launched vLLM instance. Please be explicit. Is there a wait for anything in between these two actions by the Launcher?

Reply:
You seem to be talking about something making some HTTP request to the vLLM instance. Please show it in the diagram.